tox.ini

```diff
@@ -201,13 +201,13 @@ commands = python scripts/release.py {posargs}
 
 [pytest]
 minversion = 2.0
-plugins = pytester
```

Comment:
This was added in b6ec5a57, but appears to never has been supported?!

Reply:
Hmm yeah, I can't find any code that supports this